### ISSUE TITLE
Problems building 1.0.25

### ISSUE THREAD

**Issue by lnikj** [micah-frank-studio/Lore#12]
Cabbage can run the file, and will build a standalone and plugins, but the standalone won't run and DAWs do not see the plugins.

Cabbage is latest dev: 2.8.154. OS X 12.6

--

Csound output message console

/*============================================================*/
UnifiedCSD:  /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/Lore.csd
Macro definition for omacro:GROUPBOX="GROUPBOX lineThickness(0.5), outlineThickness(0.5), colour(\"5,500,0,0\")"
Macro definition for omacro:BUTTON2="BUTTON2 fontColour:0(\"200,200,200,180\"), fontColour:1(\"0,200,0,250\"), outlineColour(\"200,200,200\"), colour:0(0,0,0), outlineThickness(2), corners(0), automatable(1)"
Macro definition for omacro:BUTTON1="BUTTON1 fontColour:0(\"250,250,250,200\"), fontColour:1(\"250,250,250\"), outlineColour(\"250,250,250\"), colour:0(0,0,0), outlineThickness(2), corners(0), automatable(1)"
Macro definition for omacro:SLIDER1="SLIDER1 trackerColour(255,255,255), textColour(255,255,255,200), trackerBackgroundColour(250,250,250,808), trackerThickness(0.05), popupText(0), _isSlider(\"yes\")"
instr Empty uses instrument number 3
instr Cassette uses instrument number 4
instr Particle uses instrument number 6
instr ImpulseVerb uses instrument number 7
instr Scrambler uses instrument number 8
instr GhostEcho uses instrument number 17
instr mixer uses instrument number 18
Elapsed time at end of orchestra compile: real: 0.019s, CPU: 0.019s
sorting score ...
	... done
Elapsed time at end of score sort: real: 0.019s, CPU: 0.019s
--Csound version 6.18 (double samples) Nov  6 2022
[commit: 7b7661e9a902473e30e98983bc2983f74e37f838]
libsndfile-1.1.0
sr = 48000.0, kr = 3000.000, ksmps = 16
0dBFS level = 1.0, A4 tuning = 440.0
WARNING: Seeding from current time 1623593685

ftable 201:
audio sr = 48000, stereo, reading both channels
opening WAV infile /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/Falkland Tennis Court.wav
ftable 202:
audio sr = 48000, stereo, reading both channels
opening WAV infile /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/Hamilton Mausoleum.wav
WARNING: GEN1: file truncated by ftable size
WARNING: 	audio samps 720000 exceeds ftsize 524288
non-deferred ftable 202 needs size 1048576
ftable 203:
audio sr = 48000, stereo, reading both channels
opening WAV infile /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/R1 Nuclear Reactor.wav
WARNING: GEN1: file truncated by ftable size
WARNING: 	audio samps 973440 exceeds ftsize 524288
non-deferred ftable 203 needs size 1048576
ftable 204:
WARNING: /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/St Georges Close.wav sr = 96000, orch sr = 48000.0
audio sr = 96000, stereo, reading both channels
opening WAV infile /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/St Georges Close.wav
ftable 205:
WARNING: /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/St Georges Far.wav sr = 96000, orch sr = 48000.0
audio sr = 96000, stereo, reading both channels
opening WAV infile /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/St Georges Far.wav
ftable 206:
WARNING: /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/St Georges Med.wav sr = 96000, orch sr = 48000.0
audio sr = 96000, stereo, reading both channels
opening WAV infile /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/St Georges Med.wav
ftable 207:
audio sr = 48000, stereo, reading both channels
opening WAV infile /Users/nikjewell/Downloads/Lore-main-1/Lore Experimental Sound Console/includes/IRs/Usina Symphony.wav
MIDI channel 1 muted
MIDI channel 2 muted
MIDI channel 3 muted
MIDI channel 4 muted
MIDI channel 5 muted
MIDI channel 6 muted
MIDI channel 7 muted
MIDI channel 8 muted
MIDI channel 9 muted
MIDI channel 10 muted
MIDI channel 11 muted
MIDI channel 12 muted
MIDI channel 13 muted
MIDI channel 14 muted
MIDI channel 15 muted
MIDI channel 16 muted
ftable 97:
ftable 98:
ftable 99:
ftable 101:
ftable 102:
ftable 103:
ftable 104:
ftable 105:
ftable 106:
ftable 107:
ftable 108:
ftable 109:
ftable 110:
ftable 1:
ftable 111:
ftable 112:
orch now loaded
audio buffered in 1024 sample-frame blocks
SECTION 1:
Cabbage version:2.8.154
new alloc for instr 1:
WARNING: Seeding from current time 1623633624

WARNING: Seeding from current time 1623633625

WARNING: Seeding from current time 1623633625

new alloc for instr 5:
new alloc for instr mixer:
  rtevent:	   T  0.086 TT  0.086 M:  0.00000  0.00000
new alloc for instr 9:
WARNING: instr 9 uses 5 p-fields but is given 6
SeffectName = 
EffectPath = includes/Effects/.orc
WARNING: instr  not found
WARNING: compileorc: could not open includes/Effects/.orc

new alloc for instr 9:
WARNING: instr 9 uses 5 p-fields but is given 6
SeffectName = 
EffectPath = includes/Effects/.orc
WARNING: instr  not found
WARNING: compileorc: could not open includes/Effects/.orc

new alloc for instr 9:
WARNING: instr 9 uses 5 p-fields but is given 6
SeffectName = 
EffectPath = includes/Effects/.orc
WARNING: instr  not found
WARNING: compileorc: could not open includes/Effects/.orc

new alloc for instr 9:
WARNING: instr 9 uses 5 p-fields but is given 6
SeffectName = 
EffectPath = includes/Effects/.orc
WARNING: instr  not found
WARNING: compileorc: could not open includes/Effects/.orc

**Comment by wpostma** [comment]
Is this another bug  report for the same underlying cabbage runtime issue (a C++ bug in cabbage, case of something was wrong) discussed elsewhere?

**Comment by wpostma** [comment]
See discussion
https://forum.cabbageaudio.com/t/help-with-compiling-lore/3275

**Comment by micah-frank-studio** [comment]
Yes, this should be resolved now in the latest Lore build with the latest public Cabbage: https://dev.azure.com/rorywalsh/cabbage/_build
